bike_rent/demo/bike_rent_demo.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
 
-<record id="product_product_bike1" model="product.product">
+<record id="product_product_bike1_demo" model="product.product">
```

Comment:
As per [OCA Guidelines](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#file-naming) not a `bike.rent` record, use separate `demo/*.xml` file, every other record that doesn't belong here too.